spec/wormbots/world_spec.rb

```diff
@@ -91,10 +92,10 @@ def world_factory(*worms)
 def worm_factory(attributes = {})
   default_attributes = {
     live: nil,
-    fertile?: true,
+    fertile?: false,
     decomposed: false,
     defertilize: nil,
     coordinates: []
   }
-  double(:worm, default_attributes.merge(attributes))
+  instance_double("Worm", default_attributes.merge(attributes))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.